### PR TITLE
PAY-74196: Add documentation for 'Continously running integration tests'

### PR DESCRIPTION
**Pull request by madfrdanskebank**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -264,6 +264,28 @@ To ensure no unauthorized calls to your callback endpoints, we strongly suggest
 ## Prefilled phone number
 You can provide a phone number to be prefilled in the phone number field on the MobilePay webpage. 
 You do this by adding an URI encoded "alias" parameter with the phone number to the search parameters of the "redirectToMPUrl". The phone number must be fully specified including country code. For "+45 12 34 56 78" you would add the following to the url: &alias=%2B4512345678
+
```
Comment (madfrdanskebank) [inline README.md:267]: Test comment?

**Review by BenjaminGraugaard — COMMENTED** [review]
can you see my comments now?

File: README.md [hunk 2]
```diff
@@ -264,6 +264,28 @@ To ensure no unauthorized calls to your callback endpoints, we strongly suggest
...
+## Continously running integration tests
+
+Mobile Pay Api now supports automated integration tests running in the test environment 'SandProd'. To support the flow from payment creation to capture, we have added a new endpoint to emulate the neccessary user behavior.
+
+The Api can be found [here](https://proxy-sandprod-az2-front-ext-rest.ext.mobilepay.dk/cardpassthrough-regressiontester-restapi/swagger/index.html?urls.primaryName=product) where the endpoint
```
[the remark this answers is not in the document]
Comment (madfrdanskebank) [reply README.md]: TODO: reference the product api when it becomes published

[the remark this answers is not in the document]
Comment (BenjaminGraugaard) [reply README.md]: it cannot be done, the url to the product API changes every time we publish it. Keeping references to it throughout this document updated will be a nightmare.

**Pull request merged**